### ISSUE TITLE
Unable to use nlu http interpreter with core model loaded in 1.0.2

### ISSUE THREAD

**Issue by Sekhar-jami** [RasaHQ/rasa#3672]
**Rasa version**: 1.0.2

**Python version**: 3.6.3

**Operating system**: osx

**Issue**: I have a core model trained & running in the 1.0.1 rasa version. And I need it to use a NLU endpoint over http (RasaNLUHttpInterpreter). I have added the endpoint in my endpoints.yml file, but the core model still keeps defaulted to regexinterpreter. The issue seems to be because the new rasa model appends 'core' & 'nlu' to the unpacked rasa model path to get the respective model paths. And since there's no 'nlu' model at the expected path it defaults to using the RegexInterpreter. (even though there's an NLU 'endpoint' available)

**Content of endpoints file (endpoints.yml)**:
```yml
nlu:
  url: http://localhost:8080/
```

**Comment by akelad** [comment]
Thanks for raising this issue, @tmbo  will get back to you about it soon.

**Comment by tmbo** [comment]
What command are you using to start the core server and what command are you using to start the nlu server?

**Comment by Sekhar-jami** [comment]
@tmbo  Here's the command I am using to start the core server (in 1.0.2):
rasa run --model ../path_to_model --endpoints ../path_to_endpoints_file --credentials ../path_to_credentials_file --enable-api

And I am running a separately hosted NLU server which is on an older version of rasa. And it is available over http at http://localhost:8080/

**Comment by Sekhar-jami** [comment]
There are also 2 interesting things in the code at rasa/core/interpreter.py:
1.  The Exception still suggests you to use the --nlu flag to specify the model to be used. But I believe this flag is no longer supported in 1.0 & above versions.

2. I also tried starting the core server with a rasa model that has both core & nlu models (in 1.0.2). But I also added the nlu endpoint in my endpoints.yml file. In this scenario, the code reaches the section that tries to create a RasaNLUHttpInterpreter. (This seems to be because of an incomplete/incorrect if condition). But since the path provided is the actual filepath to the local NLU model, even the RasaNLUHttpInterpreter is created incorrectly. The code I'm talking about is in the create method at  rasa/core/interpreter.py

**Comment by ottonemo** [comment]
I can confirm this bug - we stumbled over this as well. 

There does not seem to be a way to use the `RasaNLUHttpInterpreter`. We tried creating a dummy `nlu-%Y%m%d-%h%M%S.tar.gz` to satisfy the obscure creation logic in `NaturalLanguageInterpreter.create` but this fails as well since the unpacked dummy model path will be `/tmp/tmpABCD/nlu` which has three parts after split which in turn always leads to an exception.

As a workaround we monkey-patched `rasa.core.run.load_agent_on_start` but this is obviously not a good way forward.

Tested with rasa 1.0.6.
